coordinator/src/services/translation/response/common.ts

```diff
@@ -196,7 +196,30 @@ export function translateAgentPerson(agentPerson: hl7V3.AgentPerson): Translated
       locations
     }
   } else {
-    if (isSecondaryCare(representedOrganization)) {
+    if (prescriptionType?.startsWith("01", 0)) {
```

Comment:
Could this check be done in a function that makes it clear what is being checked? E.g. something like `isPrimaryOrSeconaryCare` - it makes it easier, for people who don't have much context, to understand what the code is doing

Reply:
`shouldHavePrimaryCareFormat`?

Reply:
Added